backend/api/tests/views/test_views_performance.py

```diff
@@ -347,3 +350,64 @@ def test_performance_export_filter_and(self):
         content_list = list(response.streaming_content)
         self.assertEqual(len(content_list), 4)
         self.assertTrue(status in str(content_list[1]))
+
+
+@pytest.fixture
```

Comment:
If we use this fixture in multiple places, maybe it's worth putting it in `conftest.py`?